src/ui/zcl_abapgit_gui_view_repo.clas.abap

```diff
@@ -252,9 +254,18 @@ CLASS zcl_abapgit_gui_view_repo IMPLEMENTATION.
       ro_toolbar->add( iv_txt = 'Tag'
                        io_sub = lo_tb_tag ) ##NO_TEXT.
     ELSE.
-      ro_toolbar->add( iv_txt = 'Import ZIP'
-                       iv_act = |{ zif_abapgit_definitions=>c_action-zip_import }?{ lv_key }|
-                       iv_opt = zif_abapgit_definitions=>c_html_opt-strong ).
+      IF mo_repo->has_remote( ) = abap_true.
+        ro_toolbar->add( iv_txt = 'Apply'
```

Comment:
Suggest just having the same "Pull" as for online repos, will it work?

Reply:
Imho no. "Apply" emphasizes that it is A) it is not online (visually different) B) this is a confirmation of some previous step. So it is more semantically correct.

Reply:
but it will trigger exactly the same code? ie. "Apply" and "Pull" will do the same thing semantically

Reply:
the discussion heads toward very theoretical matters ... ;P
If you insist I can change to Pull. But I think Apply is better. Namely because people get used to the fact that Pull is something connected with network and online. (pls confirm this or that)

Reply:
I'd just like to move it in a directly where offline projects is not different than online, this will hopefully make it break less often. It can be "Apply", just as long as it hits the same IV_ACT as "Pull"